.github/workflows/pages-html-proofer.yml

```diff
@@ -33,4 +33,4 @@ jobs:
             token: ${{ secrets.GITHUB_TOKEN }}
         - run: bundle config set --local without 'github-pages'
         - run: bundle install
-        - run: bundle exec htmlproofer ./_site/index.html --disable-external --ignore_empty_alt --allow_hash_href --allow-missing-href
+        - run: bundle exec htmlproofer ./_site/index.html --disable_external --ignore_empty_alt --allow_hash_href --allow_missing_href
```

Comment:
I recommend rebasing your feature branches with milestone branch in future to avoid these non-related commits showing up in your PR.